algorithms/CPlusPlus/Linked-Lists/remove-duplicates-in-sorted-linked-list.cpp

```diff
@@ -0,0 +1,56 @@
+/*
+Traverse the list from the head (or start) node. 
+While traversing, compare each node with its next node. 
+If the data of the next node is the same as the current node then delete the next node. 
+Before we delete a node, we need to store the next pointer of the node 
+*/
```

Comment:
Add a user-friendly description of the program 
i.e
Program to remove the duplicate value in a sorted linked list using the x method

Reply:
@UG-SEP added description